tests/testcore/functional_test_base.go

```diff
@@ -184,7 +184,7 @@ func (s *FunctionalTestBase) SetupSuite(defaultClusterConfigFile string, options
 	s.operatorClient = s.testCluster.OperatorClient()
 	s.httpAPIAddress = cluster.Host().FrontendHTTPAddress()
 
-	s.namespace = RandomizeStr("functional-test-namespace")
+	s.namespace = "functional-test-namespace"
```

Comment:
this needs to be random for parallel test suites, no?

if the name is too long then we should just make RandomizeStr use fewer chars (6 random base64 chars should be fine)

Reply:
lol yes, changed for debugging, wanted to revert before sending the PR.